### DISCUSSION TITLE
Hillshading

### DISCUSSION THREAD

**Discussion by damianofalcioni** [organicmaps/organicmaps#863] [Ideas]
A light Hillshading effect will improve the visibility of pendencies, especially at lower zoom levels, and will be perfect in association with the elevation lines.
Samples at [https://en.mapy.cz/turisticka](https://en.mapy.cz/turisticka), [https://opentopomap.org/](https://opentopomap.org) and  [https://wiki.openstreetmap.org/wiki/Hillshading](https://wiki.openstreetmap.org/wiki/Hillshading)

**Comment by n-peugnet** [comment]
I also think this would help to understand the elevation faster. I wonder if it should be there by default or maybe be part of the "Terrain" layer ?

Edit: this is a duplicate of https://github.com/organicmaps/organicmaps/discussions/576
